src/tcMenuKeyboard.cpp

```diff
@@ -52,6 +52,41 @@ void MenuEditingKeyListener::keyPressed(char key, bool held) {
 		// we are not editing, attempt to select an item using 0-9
 		menuMgr.valueChanged(fromKeyToIndex(key));
 	}
+	else if (key == 'A') {
+		MenuItem* itm = menuMgr.getCurrentMenu();
+		bool haveSelected = false;
+		while (itm != NULL) {
+			if (itm->getNext()->isActive()) {
+				itm->getNext()->setActive(false);
+				itm->setActive(true);
+				haveSelected=true;
+				break;
+			} else {
+				itm = itm->getNext();
+			}
+		}
+		if(!haveSelected){
+			menuMgr.valueChanged(0);
+		}
+		
+	}
+	else if (key == 'B') {
+		MenuItem* itm = menuMgr.getCurrentMenu();
+		bool haveSelected = false;
+		while (itm != NULL) {
+			if (itm->isActive()) {
+				itm->setActive(false);
+				itm->getNext()->setActive(true);
```

Comment:
Please can you wrap this in a null check, the item will have a NULL next. IE itm->getNext() may be NULL.